tests/models/test_tasks.py

```diff
@@ -20,8 +16,8 @@ def test_task_model_str(self):
         Test the str method on Task model
         """
         cow_price = mommy.make('tasking.Task', name="Cow prices")
-        expected = 'Cow prices - {}'.format(cow_price.pk)
-        self.assertEqual(expected, six.text_type(cow_price))
+        expected = f'Cow prices - {cow_price.pk}'
+        self.assertEqual(expected, str(cow_price))
```

Comment:
is the `str` necessary?

Reply:
This test was testing the string representation of the object. I can replace this with a call to `__str__()` though.